en/installconfig-hwe-kernels.md

```diff
@@ -58,25 +100,25 @@ perform these three configurations from the CLI.
 
 ### Default minimum kernel
 
-To set the default minimum HWE kernel for all machines visit the 'Settings' page
-and select a HWE kernel in the 'Default Minimum Kernel Version' field. Don't
+To set the default minimum kernel for all machines visit the 'Settings' page
+and select a kernel in the 'Default Minimum Kernel Version' field. Don't
 forget to click 'Save'.
 
 ![image](../media/default_min_hwe_kernel.png)
 
 ### Machine minimum kernel
 
-To set the minimum HWE kernel on a machine basis visit the machine's page and click
-the `Edit` button in the 'Machine summary' pane. Then select a HWE kernel in the
+To set the minimum kernel on a machine basis visit the machine's page and click
```

Comment:
I think the 'machine's page' should reference the Nodes page, i.e: To set the minimum kernel on a machine basis, select a machine from the Nodes page and click...